UnityClient/Assets/Scripts/TowerDefence/Win.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Win : MonoBehaviour
```

Comment:
I'm assuming this type is just a placeholder for now, and won't review it as a result this time.